engine/engine/src/engine.cpp

```diff
@@ -311,10 +311,10 @@ namespace dmEngine
         m_Register = dmGameObject::NewRegister();
         m_InputBuffer.SetCapacity(64);
         m_ResourceTypeContexts.SetCapacity(31, 64);
-
-        m_PhysicsContext.m_Context3D = 0x0; // it'a union
-        m_PhysicsContext.m_Debug = false;
-        m_PhysicsContext.m_3D = false;
+        m_PhysicsContextBox2D.m_Context = 0x0;
+        m_PhysicsContextBox2D.m_BaseContext.m_PhysicsType = dmGameSystem::PHYSICS_ENGINE_BOX2D;
+        m_PhysicsContextBullet3D.m_Context = 0x0;
+        m_PhysicsContextBullet3D.m_BaseContext.m_PhysicsType = dmGameSystem::PHYSICS_ENGINE_BULLET3D;
```

Comment:
These enums are a little bit surprising though. I wouldn't have thought we needed such info after the split?

Reply:
Nvm, I saw the code in collision object cpp 👍